### DISCUSSION TITLE
missing settings folder, and missing test notifications

### DISCUSSION THREAD

**Discussion by astro46** [wokhansoft/WFN#126] [General]
I have been using wfn for several years. installed the current wfn x64 2.5 beta standalone on win10. I am getting the missing settings folder error, which looks like has already been reported (settings json error), both when clicking reload, and options>save.

I don't see the test notifications button that was included in 2.0 version. I would be more comfortable if I could test that notification are working. I installed on another computer, and apparently wfn wasn't really notifying, because I went back to version 2.0 and several notifications popped up from when 2. 5 was running.

**Comment by aaronnabil** [comment]
I'm getting the same error.

I just created it manually, seems to be OK.
